resources/js/components/PrincipleAssessmentModal.vue

```diff
@@ -138,12 +165,17 @@ const props = defineProps({
     principleAssessment: Object,
     assessmentType: String,
     closing: Boolean,
+    // how to add props to extract form value?
```

Comment:
Is this a question for me, or a note for you to find out later?

Either way, this is something you should be able to remember pretty quickly by reviewing the other form elements in this component, or a quick Google search.

Reply:
This is a note for me to find out later.

Yes, I recall my memory after reviewing my previous study note and working on dashboard PR.